mev_inspect/aave_liquidations.py

```diff
@@ -51,7 +51,17 @@ def get_aave_liquidations(
                 trace.transaction_hash, trace.trace_address, traces
             )
 
-            received_amount = _get_liquidator_payback(child_traces, liquidator)
+            payback_transfer = _get_liquidator_payback(child_traces, liquidator)
+
+            if payback_transfer:
```

Comment:
`if payback_transfer is not None` - explicit in your expectations

one time in a project I had an error that came down to an `if x` and it turned out that we thought x was `None` but it was somehow False (the types were crazy in this repo)